### PR TITLE
Create monitor-deploy.md and tidb-monitoring-api.md

### PR DESCRIPTION
**Pull request by xiaoyangwang5**
deploy monitor

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): #3186

### CODE REVIEW

**Comment by xiaoyangwang5** [comment]
deploy monitor

File: TOC.md [hunk 1]
```diff
@@ -49,6 +49,7 @@
   + 监控与告警设置
     + [监控框架概述](/tidb-monitoring-framework.md) @李宋高
     + [监控 API](/tidb-monitoring-api.md) @李宋高
+    + [监控部署](/monitor-deploy.md) @李宋高
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply TOC.md:52]: @xiaoyangwang5 If you commit this suggestion, remember to update  the corresponding link in https://github.com/pingcap/docs-cn/pull/3186 as well, thanks~

[the remark this answers is not in the document]
Comment (xiaoyangwang5) [reply TOC.md]: done

File: monitor-deploy.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
+---
```
[the remark this answers is not in the document]
Comment (xiaoyangwang5) [reply monitor-deploy.md]: done

**Comment by TomShawn** [comment]
The change of https://github.com/pingcap/docs-cn/pull/3186 is migrated to this PR.

File: tidb-monitoring-api.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## 使用状态接口
+
+状态接口用于监控组件的一些基本信息，并且可以作为 keepalive 的监测接口。另外，通过 PD 的状态接口可以看到整个 TiKV 集群的详细信息。
+
+### TiDB Server
+
+- TiDB API 地址：`http://${host}:${port}`
+- 默认端口：10080
+- 各类 `api_name` 详细信息：参见 [TiDB API 文档](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)
```
Comment (kissmydb) [inline tidb-monitoring-api.md:22]: 这个链接不就自己？

File: tidb-monitoring-api.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+---
+title: TiDB 集群监控 API
+category: how-to
+aliases: ['/docs-cn/dev/monitor-a-tidb-cluster/']
```
Comment (kissmydb) [inline tidb-monitoring-api.md:4]: 这个应该是 api 吧

File: deploy-monitoring-services.md [hunk 5]
```diff
@@ -1,97 +1,11 @@
 ---
-title: TiDB 集群监控
+title: 集群监控部署
 category: how-to
-aliases: ['/docs-cn/dev/how-to/monitor/monitor-a-cluster/']
 ---
 
-# TiDB 集群监控
+# TiDB 集群监控部署
```
Comment (kissmydb) [inline deploy-monitoring-services.md:7]:
```suggestion

本文档适用于希望手动部署 TiDB 监控报警系统的用户。TiUP 部署方式，会同时自动部署监控报警系统，无需手动部署。
```

File: deploy-monitoring-services.md [hunk 6]
```diff
@@ -1,97 +1,12 @@
...
-Metrics 接口用于监控整个集群的状态和性能。
-
-- 如果使用 TiDB Ansible 部署 TiDB 集群，监控系统（Prometheus 和 Grafana）会同时部署。
-- 如果使用其他方式部署 TiDB 集群，在使用 metrics 接口前，需先[部署 Prometheus 和 Grafana](#部署-prometheus-和-grafana)。
-
-成功部署 Prometheus 和 Grafana 之后，[配置 Grafana](#配置-grafana)。
-
-### 部署 Prometheus 和 Grafana
+本文档适用于希望手动部署 TiDB 监控报警系统的用户。TiUP 部署方式，会同时自动部署监控报警系统，无需手动部署。
+## 部署 Prometheus 和 Grafana
```
Comment (TomShawn) [inline deploy-monitoring-services.md:9]:
```suggestion

## 部署 Prometheus 和 Grafana
```

File: tidb-monitoring-api.md (hunk 3 above)
Comment (TomShawn) [inline tidb-monitoring-api.md:22]:
```suggestion
```

File: tidb-monitoring-api.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
```
Comment (TomShawn) [inline tidb-monitoring-api.md:4]:
```suggestion
aliases: ['/docs-cn/dev/monitor-a-tidb-cluster/','/docs-cn/dev/how-to/monitor/monitor-a-cluster/']
```

**Pull request merged**